docs/running-porch-in-kind-cluster.md

```diff
@@ -0,0 +1,253 @@
+# Build and deploy custom porch images on a kind cluster
+
+This section covers the process to build and deploy the porch images to a given kind cluster.
+
+## Prerequisites
+The following software should be installed prior to running:
+1. [git](https://git-scm.com/)
+2. [Docker](https://www.docker.com/get-started/)
+Docker image load bug on v25.0.0 Use v24.0.7
+3. [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
+4. [The go programming language](https://go.dev/doc/install)
+5. [kind](https://kind.sigs.k8s.io/docs/user/quick-start#installation)
+6. [kpt-cli](https://kpt.dev/installation/kpt-cli)
+
+## Create a default kind cluster
+
+If one is not already available, create a kind cluster.
+
+```
+kind create cluster -n dev
+```
+
```

Comment:
```suggestion

Save the config for the new cluster and set it as the default config in the current shell:

```
kind get kubeconfig --name=dev > ~/.kube/kind-dev-config
export KUBECONFIG=~/.kube/kind-dev-config
```
```

Reply:
I had considered it but wanted to leave it open to the user to pass it to the make target. I'll add it as "Optionally, ..."